tekton/pipeline_run_test.go

```diff
@@ -62,16 +70,40 @@ var _ = Describe("PipelineRun", func() {
 		}
 		release = &v1alpha1.Release{
```

Comment:
I thought I commented on this but probably I forgot. Although outside of this patch, it seems that the release doesn't change per test. You could have a `BeforeAll` for it.